test/functionalTest/cases/3190_notify_only_attributes_that_change/notify_only_attributes_that_change_get_and_update_subscription.test

```diff
@@ -0,0 +1,218 @@
+}'
+orionCurl --url /v2/subscriptions --payload "$payload"
+subId=$(echo "$_responseHeaders" | grep Location | awk -F/ '{ print $4 }' | tr -d "\r\n")
+echo
+echo
+
+
+echo '02. GET /v2/subscriptions/{subscription_id}, Get subscription to check onlyChanged: true is rendered'
+echo '===================================================================================================='
+orionCurl --url /v2/subscriptions/${subId}
+echo
+echo
+
+
+echo '03. PATCH /v2/entities/Room1/attrs, update the value of A and B attribute.'
+echo '=========================================================================='
+payload='{
+    "notification": {
+    	"http": {
+           "url": "http://localhost:1028/notify"
+        },
+        "attrs": [
+           "B","C","D"
+        ],
+        "onlyChangedAttrs": false
+    }
+}'
+orionCurl --url /v2/subscriptions/${subId} --payload "$payload" -X PATCH
+echo
+echo
+
+
+echo '04. GET /v2/subscriptions/{subscription_id}, Get subscription to check onlyChanged: false is rendered'
+echo '====================================================================================================='
+orionCurl --url /v2/subscriptions/${subId} 
+echo
+echo
+
+
+--REGEXPECT--
+01. POST /v2/subscriptions, Create a subscription with onlyChanged: true.
+=========================================================================
+HTTP/1.1 201 Created
+Content-Length: 0
+Location: /v2/subscriptions/REGEX([0-9a-f]{24})
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+02. GET /v2/subscriptions/{subscription_id}, Get subscription to check onlyChanged: true is rendered
+====================================================================================================
+HTTP/1.1 200 OK
+Content-Length: 383
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "description": "A subscription to get info about Room1", 
+    "expires": "2040-01-01T14:00:00.00Z", 
+    "id": "REGEX([0-9a-f]{24})", 
+    "notification": {
+        "attrs": [
+            "B", 
+            "C", 
+            "D"
+        ], 
+        "attrsFormat": "normalized", 
+        "http": {
+            "url": "http://localhost:1028/notify"
+        }, 
+        "onlyChangedAttrs": true
+    }, 
+    "status": "active", 
+    "subject": {
+        "condition": {
+            "attrs": [
+                "A", 
+                "B", 
+                "C"
+            ]
+        }, 
+        "entities": [
+            {
+                "id": "Room1", 
+                "type": "Room"
+            }
+        ]
+    }, 
+    "throttling": 5
+}
+
+
+03. PATCH /v2/entities/Room1/attrs, update the value of A and B attribute.
+==========================================================================
+HTTP/1.1 204 No Content
+Content-Length: 0
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+04. GET /v2/subscriptions/{subscription_id}, Get subscription to check onlyChanged: false is rendered
```

Comment:
Actually it seems that onlyChange: false is not rendered...

Reply:
fixed in 1a364d6